engine/language-server/src/main/scala/org/enso/languageserver/libraries/handler/LibraryGetMetadataHandler.scala

```diff
@@ -5,7 +5,7 @@ import akka.actor.{Actor, ActorRef, Cancellable, Props, Status}
 import scala.util.{Success, Try}
 import akka.pattern.pipe
 import com.typesafe.scalalogging.LazyLogging
-import nl.gn0s1s.bump.SemVer
+import com.github.zafarkhaja.semver.Version
```

Comment:
How do you find these dependencies? `zafarkhaja` seems like a name from a fairy tale.

Reply:
I believe it is developer's name

Reply:
I ditched the dependency.